doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -205,18 +205,15 @@ In Keystone v1 and v2 (both deprecated), the term "tenant" was used in OpenStack
 OpenStack offers multi-tenancy by means of resource (compute, network and storage)separation via projects. OpenStack offers ways to share virtual resources between projects while maintaining logical separation. As an example, traffic separation is provided by creating different VLAN ids for neutron networks of different projects. As another example, if host separation is needed, nova scheduler offers AggregateMultiTenancyIsolation scheduler filter to separate projects in host aggregates. Thus, if a host in an aggregate is configured for a particular project, only the instances from that project are placed on the host. Overall, tenant isolation ensures that the resources of a project are not affected by resources of another project.
 
 <a name="3.3.3"></a>
-### 3.3.3. Host aggregates providing resource pooling
-Availability zones: provide resiliency and fault tolerance for VM deployments, by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms
+### 3.3.3. Cloud partitioning: Host Aggregates, Availability Zones
+Cloud administrators can partitioned the hosts within an OpenStack cloud using 2 mechanisms: Host Aggregates and Availability Zones.
 
-Host aggregate: is a Cloud Admin concept which is used to map the VNFC instances on the compute nodes
+A Host Aggregate is a group of hosts (compute nodes) with specific characteristics, software and/or hardware properties. Example would be a Host Aggregate created for specific hardware or performances. The administrator assign key-value pairs to groups of compute nodes, this information will be used to schedule VMs. A host can belong to multiple host aggregates. Host Aggregates are not explicitly exposed to tenants.
```

Comment:
"The administrator assign key-value pairs to groups of compute nodes, this information will be used to schedule VMs." -- I think the key-value properties are assigned to the host aggregates.

Typo: ""The administrator assign ..." ---> "The administrator assigns ..."

Reply:
@pgoyal01 , Can we mention "with specific characteristics with the same specification"

Reply:
@pgoyal01 , we have a deployment, one HA for DPDK workload, 2nd HA for ovs deployment ,...

Reply:
@goyal I'm not sure to catch your point. For me "The administrator assign key-value pairs to groups of compute nodes" is equivalent to "key-value properties are assigned to the host aggregates".
Typo fixed

@ASawwaf Yes, I can add with the same specification
-> done in commit 3

Reply:
@karinesevilla I leave it up to you to decide whether to use "groups of compute nodes" or "host aggregate". Thanks

Reply:
@pgoyal01 Got it. I change the sentence if it's clearer.